DATA_PREPARATION.md

```diff
@@ -200,15 +200,21 @@ For `key_value_pair` dataset, an additional field `schema` is required to define
-| examples    | list[string]                      | examples of field content                                                                                                                                      | no                      |
-| enum        | list[string \| number \| integer] | only work when type is string or number or integer                                                                                                             | no                      |
-| properties  | dict[string, FieldSchema]         | define FieldSchema of each subfield                                                                                                                            | yes when type is object |
-| items       | FieldSchema                       | define the FieldSchema for all items in array                                                                                                                  | yes when type is array  |
-
-More details can be found at [`vision-datasets/vision_datasets/key_value_pair/manifest.py`](vision_datasets/key_value_pair/manifest.py)
+| Property         | Type                      | Example                                                                                                     | Required?               |
+| :--------------- | :------------------------ | :---------------------------------------------------------------------------------------------------------- | :---------------------- |
+| type             | FieldValueType            | JSON type: string, number, integer, boolean, array, object.                                                 | yes                     |
+| description      | string                    | answer to question                                                                                          | no                      |
+| examples         | list[string]              | examples of field content                                                                                   | no                      |
+| classes          | dict[str, ClassSchema]    | dictionary that maps each class name to `ClassSchema`.                                                      | no                      |
+| properties       | dict[string, FieldSchema] | define FieldSchema of each subfield                                                                         | yes when type is object |
+| items            | FieldSchema               | define the FieldSchema for all items in array                                                               | yes when type is array  |
+| includeGrounding | boolean                   | Whether annotation of this field contains bbox groundings, if true, bboxes are stored in `groundings` field | No, default false       |
+
+Definition of `ClassSchema`:
+| Property    | Type   | Example                          | Required?         |
```

Comment:
For consistency, maybe `Example` -> `Details` and change the content of the first row to `describes the class in more detail, e.g., "long, thin, surface-level mark"`?